common/cluster/metadata.go

```diff
@@ -107,8 +110,9 @@ type (
 		// Address indicate the remote service address(Host:Port). Host can be DNS name.
 		RPCAddress string `yaml:"rpcAddress"`
 		// Cluster ID allows to explicitly set the ID of the cluster. Optional.
-		ClusterID  string `yaml:"-"`
-		ShardCount int32  `yaml:"-"` // Ignore this field when loading config.
+		ClusterID  string            `yaml:"-"`
+		ShardCount int32             `yaml:"-"` // Ignore this field when loading config.
+		Tags       map[string]string `yaml:"-"` // Ignore this field. Use cluster.Config.Tags for customized tags.
```

Comment:
if ignored, why still define it here?

Reply:
We use this struct in memory in all places. I tried to minimize the change in this PR and will do better in https://github.com/temporalio/temporal/pull/4620